brother_ql/reader.py

```diff
@@ -66,9 +66,12 @@
 
 RESP_MEDIA_TYPES = {
   0x00: 'No media',
-  0x01: 'Laminated Tape',
+  0x01: 'Laminated tape',
+  0x03: 'Non-laminated tape',
   0x0A: 'Continuous length tape',
   0x0B: 'Die-cut labels',
+  0x11: 'Heat-shrink tube',
+  0xff: 'Incompatible tape',
```

Comment:
Question: Is this directly related to network printing? If so how?

Reply:
This isn't directly related to network printing... while I like the structure to be complete, it has no real place in this PR. Will revert that change also